src/main/java/org/deepsymmetry/beatlink/VirtualCdj.java

```diff
@@ -1418,6 +1456,18 @@ private void deliverDeviceUpdate(final DeviceUpdate update) {
         }
     }
 
+    // This is a way to get mediaDetails from Rekordbox and pass to all MediaDetailsListeners
+    private final MediaDetailsListener mediaDetailsListener = new MediaDetailsListener(){
+        @Override
+        public void detailsAvailable(MediaDetails details) {
+            deliverMediaDetailsUpdate(details);
+        }
+    };
+
+    public MediaDetailsListener getMediaDetailsListener() {
```

Comment:
This is immutable, so there is no harm in making the field itself public rather than implementing a getter, right?